asps/Simulation/starsim/Conscript

```diff
@@ -90,7 +90,7 @@ $CPPPATH .= $main::PATH_SEPARATOR . "#asps/Simulation/starsim/include";
 $CPPPATH .= $main::PATH_SEPARATOR . $env->{Packages}->{MYSQL}->{INCDIR}; 
 $CPPPATH .= $main::PATH_SEPARATOR . $env->{ENV}->{CERN_ROOT} . "/include";
 $CPPPATH .= $main::PATH_SEPARATOR . $env->{ENV}->{CERN_ROOT} . "/src/pawlib/paw/ntuple";
-$CPPFLAGS  = "-DCERNLIB_DZDOC -DCERNLIB_NONEWL -DCERNLIB_SHL -DCERNLIB_HADRON -DCERNLIB_BSLASH -DCERNLIB_HIGZ -DCERNLIB_CG";
+$CPPFLAGS  = "-fcommon -DCERNLIB_DZDOC -DCERNLIB_NONEWL -DCERNLIB_SHL -DCERNLIB_HADRON -DCERNLIB_BSLASH -DCERNLIB_HIGZ -DCERNLIB_CG";
```

Comment:
`CPP` is the C Pre-Processor. This has to go to `CFLAGS` and, possibly, also `FFLAGS`.

Reply:
See https://github.com/star-bnl/star-sw/pull/370/commits/7d5fd7f3ed2a7e6b3d48d140334e6d4af5a0f2a0